main.go

```diff
@@ -48,9 +50,27 @@ func main() {
 	}
 
 	if autoCreate {
-		if _, err := os.Stat(confFile); err != nil && os.IsNotExist(err) {
-			logger.Warn("No conf file, creating one", "confFile", confFile)
+		shouldCreateOrUpdate := false
+		_, err := os.Stat(confFile)
+		if  err != nil {
+			if os.IsNotExist(err) {
+				logger.Warn("No conf file, creating one", "confFile", confFile)
+				shouldCreateOrUpdate = true
+			}
+		} else {
+			file, errOpen := os.ReadFile(confFile)
+			if errOpen != nil {
+				logger.Error("Cannot open config file", "err", errOpen)
+				return
+			}
 
+			if bytes.Equal(md5.New().Sum(file), md5.New().Sum(deprecatedConfFileContent())) {
```

Comment:
Instead of storing the deprecated file content in a var, why not store the md5 hash itself?

You could leave the deprecated file in a comment for posterity.

Reply:
Good idea. I'll test this out once and use has if everything is good.